linkerd/proxy/http/src/upgrade.rs

```diff
@@ -139,7 +139,7 @@ impl Drop for Inner {
             let both_upgrades = async move {
                 let (server_conn, client_conn) = tokio::try_join!(server_upgrade, client_upgrade)?;
                 trace!("HTTP upgrade successful");
-                if let Err(e) = Duplex::new(server_conn, client_conn).await {
+                if let Err(e) = Duplex::new(client_conn, server_conn).await {
```

Comment:
these had to be reversed to get the correct direction labels, which, admittedly, is not the best. I also considered passing each label in as an argument (which feels a little confusing since there's now 4 positionals), or making `HalfDuplex` public and making the constructor take two `HalfDuplex`es so that the user adds the direction labels. but, those are both a bit more complicated and since `Duplex::new` is only called in two places, i dunno how much this matters, really...